tutorials/snake/steps/step-3.md

```diff
@@ -29,7 +29,7 @@ The fruit, however, occupies a single cell - so we just need one set of coordina
     (int2 4 4) >= .fruit
     ```
 
-To simplify the layout we will render the grid using a <a href="https://docs.fragcolor.xyz/blocks/GUI/Table/" target="_blank">`GUI.Table`</a>. We just need to push each value to the next column and if there's no more space in that row a new row will be added automatically.
+To simplify the layout we will render the grid using a [`(GUI.Table)`](https://docs.fragcolor.xyz/blocks/GUI/Table/). We just need to push each value to the next column and if there's no more space in that row a new row will be added automatically.
```

Comment:
We use ` `` ` here because `(GUI.Table)` is code.

Reply:
Alright, thanks. Noted for the future.